macros/edr/tests/test_utils/backfill_result_rows.sql

```diff
@@ -0,0 +1,35 @@
+{% macro backfill_result_rows() %}
+    {% if is_incremental() %}
+        {% do return('') %}
+    {% endif %}
+
+    {% set elementary_test_results_relation = ref("elementary", "elementary_test_results") %}
+    {% set result_row_exists = elementary.get_column_in_relation(elementary_test_results_relation, "result_rows") is not none %}
+    {% if not result_row_exists %}
+        {% do return('') %}
+    {% endif %}
+
+    {% set test_result_rows_rows = [] %}
+    {% set backfill_query %}
+        select
+        id,
+        detected_at,
+        result_rows
+        from {{ ref("elementary", "elementary_test_results") }}
+        where {{ elementary.datediff(elementary.cast_as_timestamp('detected_at'), elementary.current_timestamp(), 'day') }} < {{ elementary.get_config_var("days_back") }}
+        and result_rows is not null
```

Comment:
Theoretically we should also query rows with `null` in `result_rows` and query their `test_results_query`.
However, there's a high likelihood that the test will no longer yield the same results, or even potentially crash.
In that regard, I prefer to exclude that case.